crates/fuzzing/src/oracles/diff_wasmi.rs

```diff
@@ -117,10 +92,23 @@ impl DiffEngine for WasmiEngine {
                 None => return false,
             },
         };
-        match trap.as_code() {
-            Some(wasmi::core::TrapCode::StackOverflow) => true,
-            _ => false,
-        }
+        matches!(trap.as_code(), Some(wasmi::core::TrapCode::StackOverflow))
+    }
+}
+
+/// Converts `wasmi` trap code to `wasmtime` trap code.
+fn wasmi_to_wasmtime_tap_code(trap: wasmi::core::TrapCode) -> wasmtime::TrapCode {
```

Comment:
I just noticed, I think you meant to call this function "trap_code" rather than "tap_code".

Reply:
good find! fixed it